### ISSUE TITLE
[BUG] [Formatter] html strings inside html attributes not properly formatted

### ISSUE THREAD

**Issue by oliverhaas** [djlint/djLint#1000]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/djlint/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info

- OS: Ubuntu 24.04
- Python Version: 3.12
- djLint Version: 1.35.4
- template language: django

As a note, all system information is not really relevant, since it's just straight up djlint behavior/bug.

## Issue

djlint does not respect whether an html tag is an actual html tag or just supposed to be a string inside an html attribute. This affects several libraries (e.g. I'm using hyperscript and preline), which use such "html tag strings" inside html attributes.

While one might argue that formatting html inside attributes might be nice, it currently usually breaks the formatting, rather than actually producing sensible formatting. I wouldn't know how to produce clean formatting in most or all use cases, so I think it should just be turned off.

ALSO: I think this issue is separate from https://github.com/djlint/djLint/issues/723, since that one would be rather a feature on top of this one. But I actually might have a go at that one after this since it's relevant for my use cases, too.

## How To Reproduce

Try something like

```html
<select data-html="<div></div>" data-normal="hello"></select>

<!-- expected --!>
<select data-html="<div></div>" data-normal="hello"></select>

<!-- actual --!>
<select data-html="<div>
</div>
" data-normal="hello"></select>
```

Best to look at the tests which I will link in a PR in a minute :).
